src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -15,20 +15,21 @@ public AllInCommand(IntakeSubsystem intakeSubsystem) {
 	public void initialize() {
 		intakeSubsystem.intakeIn();
 		intakeSubsystem.indexIn();
-		intakeSubsystem.feederIn();
+		// intakeSubsystem.feederIn();
 	}
 
 	@Override
 	public void execute() {
 		if (intakeSubsystem.getIndexSensor()) {
-			intakeSubsystem.intakeStop();
+			intakeSubsystem.intakeReverse();
```

Comment:
Instead of doing reverse, it would be good to have a reject that runs at a lower speed that doesn't use as much power and reduces the chance of overheating the motors.